sdk-docs/getting-started/extension-points.md

```diff
@@ -196,3 +192,24 @@ gridOptions: {
 }
 ```
 
+# Asset Events Actions extension point
+
+This extension point allows you to add a user interface in the Events within Asset Home in Trackunit Manager. The extension is controlled in the extension-manifest.ts file.
+
```

Comment:
An image for this would be great! 
But it can also wait :)

Reply:
I agree, Thank you. My plan to add an image once my extension PR gets approved :)